### DISCUSSION TITLE
[Feature Request] Play archives

### DISCUSSION THREAD

**Discussion by kswt** [PaulWoitaschek/Voice#1402] [Ideas]
Many sites distribute audio books via ZIP archives. User must unpack it before listening. Some  text e-book readers can open files in archives. It will be useful feature in Voice.

**Comment by test2a** [comment]
Vlc player does that

**Comment by atomicwrites** [comment]
I would also find this useful, in my case because both Calibre and ebook-tools don't understand books made of multiple files, so you have to zip ebooks to be able to add them to your library. It would be nice to play them later without unzipping.
